src/Stratis.Bitcoin.Features.MemoryPool/MempoolFeature.cs

```diff
@@ -141,8 +141,9 @@ public static class FullNodeBuilderMempoolExtension
         /// Include the memory pool feature and related services in the full node.
         /// </summary>
         /// <param name="fullNodeBuilder">Full node builder.</param>
+        /// <param name="injectRules">Whether or not to inject the mempool rules now, or defer it to another feature.</param>
         /// <returns>Full node builder.</returns>
-        public static IFullNodeBuilder UseMempool(this IFullNodeBuilder fullNodeBuilder)
+        public static IFullNodeBuilder UseMempool(this IFullNodeBuilder fullNodeBuilder, bool injectRules = true)
```

Comment:
uchh what an ugly hack :(

Reply:
This is only really here to cater for specific nodes that we use for testing, right @zeptin ?

I don't think this is too bad, we can almost kill all of those and just make new networks for them...